new-plans/implementation.md

```
@@ -446,40 +446,150 @@ impl Continuation {
+    let module = parse_and_lower(db, source);
+    let module = resolve(db, module);
+    let module = typecheck(db, module);
+    let module = lambda_lift(db, module);
+    // ...
+}
 ```
-Tribute Source
+
+### 파이프라인 구조
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)